iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DirectMethodsCommon.java

```diff
@@ -255,7 +363,57 @@ protected void invokeMethodSucceed() throws Exception
         // Assert
         assertNotNull(result);
         assertEquals((long)METHOD_SUCCESS, (long)result.getStatus());
-        assertEquals(METHOD_LOOPBACK + ":" + PAYLOAD_STRING, result.getPayload());
+        assertEquals(METHOD_LOOPBACK + ":" + PAYLOAD_STRING, result.getPayloadAsJsonString());
+    }
+
+    protected void invokeHelper(Object payload) throws Exception
+    {
+        DirectMethodRequestOptions options =
```

Comment:
Formatting in this file is inconsistent, are tabs supposed to be 4 spaces or 8? also there are multiple long lines in the file that can be broken off. ex: line 379

Reply:
corrected